_includes/header.html

```diff
@@ -1,76 +1,80 @@
 
 <header role="banner" class="site-header">
-
 	<a href="/" class="site-brand">
 		<img src="/resources/brand/logo.svg" alt="" class="site-brand__logo"/>
 		<span class="site-brand__name">Pa11y</span>
 	</a>
-	<div class="site-navigation" aria-label="Main Navigation">
-		<nav role="navigation">
-			<ul class="site-navigation__menu clearfix" tabindex="0" role="menubar" aria-controls="st1" >
 
-				<li class="site-navigation__item {% if page.url == '/' %}site-navigation__item--current{% endif %}" >
-					<a href="/" class="site-navigation__item-link">
+	<div class="site-nav" aria-label="Main Navigation">
+		<nav role="navigation">
```

Comment:
the aria label would better on the `<nav>` element (since that's what it's labelling)

Reply:
Actually..I'm not even sure that `<div>` is required....